rdtools/test/filtering_test.py

```diff
@@ -73,5 +73,11 @@ def test_clip_filter_upper(self):
         self.assertTrue((expected_result == filtered).all())
 
 
+def test_normalized_filter_default():
```

Comment:
Suggest updating this test or adding another that would fail if we change the default behavior. Perhaps something like: 
`pd.testing.assert_series_equal(normalized_filter(pd.Series([0.01 - 1e-16, 0.01 + 1e-16, 1e308])), pd.Series([False, True, True]))`

Reply:
Good idea, done.